internal/proto/controller/api/resources/workers/v1/worker.proto

```diff
@@ -85,3 +85,17 @@ message Worker {
   // Output only. The available actions on this resource for the requester.
   repeated string authorized_actions = 300 [json_name = "authorized_actions"]; // @gotags: `class:"public"`
 }
+
+message CertificateAuthority {
+  // Output only. The ID of the certificate
+  string id = 10; // @gotags: `class:"public"`
+
+  // Output only. The public key of the cert authority
+  bytes public_key = 20; // @gotags: `class:"public"`
+
+  // Output only. The time before which this CA is invalid
+  google.protobuf.Timestamp not_before_time = 30; // @gotags: `class:"public"`
+
+  // Output only. The time after which this CA is invalid
+  google.protobuf.Timestamp not_after_time = 40; // @gotags: `class:"public"`
+}
```

Comment:
I'd like feedback on these fields when returning a CertAuthority.
Initially had the cert DER- is that needed?